src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx

```diff
@@ -0,0 +1,1115 @@
+                                    ...base,
+                                    color: 'var(--N400)',
+                                    padding: '0 8px',
+                                    transition: 'all .2s ease',
+                                    transform: state.selectProps.menuIsOpen ? 'rotate(180deg)' : 'rotate(0deg)',
+                                }
+                            },
+                            valueContainer: (base, state) => {
+                                return {
+                                    ...base,
+                                    padding: '0 8px',
+                                }
+                            },
+                            loadingMessage: (base) => {
+                                return {
+                                    ...base,
+                                    color: 'var(--N600)',
+                                }
+                            },
+                            noOptionsMessage: (base) => {
+                                return {
+                                    ...base,
+                                    color: 'var(--N600)',
+                                }
+                            },
+                        }}
+                    />
+                </div>
+                {repoChartValue.deprecated && (
+                    <div className="chart-deprecated-wrapper flex top left br-4 cn-9 bcy-1 mt-12">
+                        <div className="icon-dim-16 mr-10">
+                            <Error className="icon-dim-16 chart-deprecated-icon" />
+                        </div>
+                        <span className="chart-deprecated-text fs-12 fw-4">
+                            This chart has been deprecated. Please select another chart to continue receiving updates.
+                        </span>
+                    </div>
+                )}
+                {isExternal && !installedAppInfo && !repoChartValue.chartRepoName && (
+                    <div className="no-helm-chart-linked flex top left br-4 cn-9 bcr-1 mt-12">
+                        <div className="icon-dim-16 mr-10">
+                            <Error className="icon-dim-16" />
+                        </div>
+                        <span className="no-helm-chart-linked-text fs-12 fw-4 cn-9">
+                            This app is not linked to a helm chart. Select a helm chart to keep up with latest chart
+                            versions.
+                        </span>
+                    </div>
+                )}
+            </div>
+        )
+    )
+}
+
+export const ChartVersionSelector = ({
+    selectedVersion,
+    chartVersionObj,
+    selectedVersionUpdatePage,
+    handleVersionSelection,
+    chartVersionsData,
+}: ChartVersionSelectorType) => {
+    return (
+        <div className="w-100 mb-12">
+            <span className="form__label fs-13 fw-4 lh-20 cn-7">Chart Version</span>
+            <Select
```

Comment:
Suggestion: We should try to change all the select to ReactSelect for the symmetry.

Reply:
Yes. Let's plan to update it at different places and remove the legacy custom select comp. 
As for this, I'm re-using the existing component so didn't update it.